packages/pds-ember/app/styles/pds/components/button/ghost/_config.scss

```diff
@@ -19,12 +19,12 @@ $HOVER: map.merge($DEFAULT, (
 ));
 /// :enabled:focus
 $FOCUS: (
-  boxShadow: theme.getShadow($focus: true),
```

Comment:
NOTE: While `PDS.Theme#getShadow()` was replaced, it does not constitute as a breaking change, because the function has only been used internally. (HCP doesn't use the function outside of the old PDS addon.)